athenahealth/httpclient.go

```diff
@@ -254,6 +252,7 @@ func (h *HTTPClient) request(ctx context.Context, method, path string, body io.R
 	if err != nil {
 		return res, err
 	}
+	defer res.Body.Close()
```

Comment:
any error between here and [line 281](https://github.com/eleanorhealth/go-athenahealth/pull/109/files#diff-bc15f4d80970f6dc182e5a8144c27e27d7f6d3facbbb8f3af3909e0ee5f617b6R281) is currently not having the request body close which is leaky.